libs/pipeline.py

```diff
@@ -25,35 +29,19 @@ class BadFipsWarning(UserWarning):
 
 def fips_to_location_id(fips: str) -> str:
```

Comment:
I noticed there were a couple other use cases of this which are calling this function `.apply(fips_to_location_id)` I think it's worth converting those calls to map rather than apply - seemed faster locally